### PR TITLE
fix(dependencies) Update jspm and systemjs-builder generated value

### LINKED ISSUE #34: Unable to install jspm dependency
Npm install crash when install jspm

```console
npm ERR! git rev-list -n1 6c85e03c9bfee63bf17408b483b15b444b7f1ada: fatal: bad object 6c85e03c9bfee63bf17408b483b15b444b7f1ada
```

Node Version: 4.4.4
Npm Version: 3.9.2

Yeoman Generator Choices: 
```console
? Which JavaScript framework do you want? React
? Which module management do you want? SystemJS with JSPM
? Which JS preprocessor do you want? ES2015 today with Babel
? Which CSS preprocessor do you want? SASS
? Do you want a sample app? A working landing page
```

I reproduce this problem with Angular2 as JavaScript framework

[EDIT] 

Use **"jspm": "^0.17.0-beta.15"** instead of **"jspm": "jspm/jspm#6c85e03c9bfee63bf17408b483b15b444b7f1ada"** correct the problem.

JSPM team released 0.17.0-beta.15 today https://github.com/jspm/jspm-cli/releases/tag/0.17.0-beta.15

### PR DESCRIPTION
**Pull request by ghoullier**
Update generated devDependencies version to the latest value to fix #34

### CODE REVIEW

**Comment by zckrs** [comment]
:+1:

File: generators/app/index.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ module.exports = fountain.Base.extend({
 
       this.mergeJson('package.json', {
         devDependencies: {
-          'jspm': 'jspm/jspm#6c85e03c9bfee63bf17408b483b15b444b7f1ada',
-          'systemjs-builder': '^0.15.15',
+          'jspm': '^0.17.0-beta.15',
```
Comment (zckrs) [inline generators/app/index.js]: Can you freeze version?

Comment (ghoullier) [inline generators/app/index.js]: Done with https://github.com/FountainJS/generator-fountain-systemjs/pull/35/commits/be4fd702838860ab1d5804cef820f06dfd57122d

**Comment by Swiip** [comment]
Perfect! Thanks

**Pull request merged**